cmd/snap/cmd_snap_op.go

```diff
@@ -113,8 +114,15 @@ type cmdInstall struct {
 }
 
 func (x *cmdInstall) Execute([]string) error {
+	var uuid string
+	var err error
+
 	cli := Client()
-	uuid, err := cli.InstallSnap(x.Positional.Snap, x.Channel)
+	if osutil.FileExists(x.Positional.Snap) {
```

Comment:
Can we please use instead `strings.Contains(x.Positional.Snap, "/")` as the test?

This has the important quality of not changing behavior depending on what sits in CWD.